examples/experimental/dagster-airlift/examples/tutorial-example/README.md

```diff
@@ -272,50 +272,51 @@ _Note: There will be some delay between task completion and assets materializing
 
 ## Migrating Assets
 
-Once you have created corresponding definitions in Dagster to your Airflow tasks, you can begin to selectively migrate execution of some or all of these assets to Dagster.
+Once you have created corresponding definitions in Dagster to your Airflow tasks, you can begin to selectively "proxy" execution of some or all of these assets to Dagster while airflow is still controlling scheduling and orchestration.
```

Comment:
I think it is important to phrase proxying in terms of tasks, because that it was actually being proxied.

"You proxy execution to Dagster on a per-Airflow-task basis."